resources/profiles/Ratrig/machine/RatRig V-Minion.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "machine_model",
+    "name": "RatRig V-Minion",
+    "model_id": "V-Minion",
+    "nozzle_diameter": "0.4",
+    "machine_tech": "FFF",
+    "family": "RatRig_V-Minion",
+    "bed_model": "ratrig-vminion-bed.stl",
```

Comment:
bed_model creates z-fighting issues.
Let's avoid using them.

Reply:
The bed model was removed due to a known z bug